### DISCUSSION TITLE
Respect install tag with user input.

### DISCUSSION THREAD

**Discussion by killagu** [npm/rfcs#604] [RRFC]
As the https://github.com/npm/cli/issues/5052 describe.

When we install npm pack with tag.
```shell
# npm i antd@conch
```

npm will use the input tag to find version and save `"antd": "^${version}"` package.json .

But actually we want save it wih `"antd": "conch"`, so after install we should modify the package.json manually.

It's better if npm auto save the pack with input tag.

And i'm pleasure to raise a pr to implement this feature.

**Comment by ljharb** [comment]
See https://github.com/npm/rfcs/pull/547

**Reply by killagu** [reply]
wow, so good!
